internal/kibana/role.go

```diff
@@ -454,13 +597,42 @@ func flattenKibanaRoleIndicesData(indices *[]kbapi.KibanaRoleElasticsearchIndice
 	return make([]interface{}, 0)
 }
 
+func flattenKibanaRoleRemoteIndicesData(indices *[]kbapi.KibanaRoleElasticsearchRemoteIndice) []interface{} {
```

Comment:
Same thing here, I don't think we're getting any value out of the pointer typed param.